### PR TITLE
cksum: Adding the functionality for --zero.

### PR DESCRIPTION
**Pull request by orangeng**
Adding the functionality for --zero / -z. As mentioned in issue #6178

### CODE REVIEW

File: src/uu/cksum/src/cksum.rs [hunk 1]
```diff
@@ -216,50 +217,58 @@ where
...
-                    println!(
+                    print!(
                         "{} ({}) = {sum}",
                         options.algo_name.to_ascii_uppercase(),
                         filename.display()
                     );
                 }
             }
         }
+        match options.zero {
```
Comment (sylvestre) [inline src/uu/cksum/src/cksum.rs:264]: please use a if
no need for match here

**Comment by sylvestre** [comment]
could you please add a test?
thanks

File: src/uu/cksum/src/cksum.rs [hunk 2]
```diff
@@ -216,50 +217,55 @@ where
...
                         options.algo_name.to_ascii_uppercase(),
                         filename.display()
                     );
                 }
             }
         }
+        if options.zero {
+            print!("\0");
+        } else {
+            print!("\n");
```
Comment (sylvestre) [inline src/uu/cksum/src/cksum.rs:267]:
```suggestion
            println!();
```

**Comment by tertsdiepraam** [comment]
Could you check out some of the other utils. I believe we added some functionality for this in uucore. I think it was called `LineEnding`.

**Comment by orangeng** [comment]
Changed to use LineEnding. As a general question, is there a good way to know if a needed code function is found in uucore and what is not?

**Comment by sylvestre** [comment]
is it ready to be reviewed? Thanks

**Comment by orangeng** [comment]
Unfortunately, school got busy and had to put this aside temporarily. Will be back on this in 1 week.

**Comment by RenjiSann** [comment]
Ping  :) @orangeng Do you have time to finish this ?

**Comment by orangeng** [comment]
@RenjiSann thanks for reminding me haha, I unfortunately don't have time so I will close this PR. Feel free to have a go at it

**Pull request closed without merging**